### PR TITLE
fix(tts): Crash fix when incorrect zero shot audio is passed

### PR DESCRIPTION
**Pull request by atomer-nvidia**

### CODE REVIEW

**Comment by rmittal-github** [comment]
@atomer-nvidia please rebase on top of `release/2.15.0` branch

File: riva/clients/tts/riva_tts_client.cc [hunk 1]
```diff
@@ -137,7 +137,13 @@ main(int argc, char** argv)
   if (not FLAGS_zero_shot_audio_prompt.empty()) {
     auto zero_shot_data = request.mutable_zero_shot_data();
     std::vector<std::shared_ptr<WaveData>> audio_prompt;
-    LoadWavData(audio_prompt, FLAGS_zero_shot_audio_prompt);
+    try {
+      LoadWavData(audio_prompt, FLAGS_zero_shot_audio_prompt);
+    }
+    catch (const std::exception& e) {
+      std::cerr << "Unable to load audio file: " << e.what() << std::endl;
```
Comment (virajkarandikar) [inline riva/clients/tts/riva_tts_client.cc:144]: use LOG(ERROR)

Comment (virajkarandikar) [inline riva/clients/tts/riva_tts_client.cc]:
```suggestion
      LOG(ERROR) << "Unable to load audio file: " << e.what() << std::endl;
```

**Comment by atomer-nvidia** [comment]
> @atomer-nvidia please rebase on top of `release/2.15.0` branch

Done

File: riva/clients/tts/riva_tts_client.cc (hunk 1 above)
Comment (atomer-nvidia) [reply riva/clients/tts/riva_tts_client.cc]: Done

**Comment by atomer-nvidia** [comment]
@virajkarandikar Is this good to merge?

**Review by virajkarandikar — APPROVED** [review]

**Pull request merged**